src/paperwork/frontend/mainwindow/__init__.py

```diff
@@ -1644,6 +1646,7 @@ def _do_import(self, file_uris):
             lambda _, stats, doc: GLib.idle_add(self.__import_ok, stats)
         )
         self.__main_win.schedulers['main'].schedule(job_importer)
+        self.file_uris = file_uris
```

Comment:
I am not sure how safe it is to define an object attribute outside ``_init_()`` ...